packages/@okta/vuepress-site/docs/release-notes/2021/index.md

```diff
@@ -3,6 +3,17 @@ title: Okta API Products release notes 2021
 ---
 ## December
 
+### Weekly release 2021.12.1
+
+| Change                                                                   | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Okta Sign-In Widget, version 5.14.1](#okta-sign-in-widget-version-5-14-1) | December 15, 2021 |
+
+#### Okta Sign-In Widget, version 5.14.1
+
+See the [Okta Sign-In Widget Release Notes](https://github.com/okta/okta-signin-widget/releases/tag/okta-signin-widget-5.14.1) for more information about this release. See the [Okta Sign-In Widget guide](/code/javascript/okta_sign-in_widget/) for more information about the Widget. <!--OKTA-452958-->
```

Comment:
```suggestion
See the [Okta Sign-In Widget Release Notes](https://github.com/okta/okta-signin-widget/releases/tag/okta-signin-widget-5.14.1) for more information on this release. See the [Okta Sign-In Widget guide](/code/javascript/okta_sign-in_widget/) for more information on the Widget. <!--OKTA-452958-->
```

Reply:
I'm ok with your suggestion, but the original blurb was pre-approved in the last release. I'm going to confirm with style/rn admins before I update this.

Reply:
If it's originally approved, keep it as is then.